File: 2.0/java/texts/getTextBroadcastBatches.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.campaigns.model.Batch;
import com.callfire.api.client.api.common.model.Page;
import com.callfire.api.client.api.common.model.request.GetByIdRequest;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        GetByIdRequest request = GetByIdRequest.create()
            .id(11646003L)
            .offset(0L)
            .limit(10L)
            .fields("items(name,status,size,remaining)")
            .build();
        Page<Batch> batches = client.textBroadcastsApi().getBatches(request);
    }
}

File: 2.0/csharp/texts/getTextBroadcastBatches.cs
using CallfireApiClient;
using CallfireApiClient.Api.Common.Model.Request;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var request = new GetByIdRequest
        {
            Id = 11646003,
            Offset = 0,
            Limit = 10,
            Fields = "items(name,status,size,remaining)"
        };
        var batches = client.TextBroadcastsApi.GetBatches(request);
    }
}
